### PR TITLE
brace expansion section

### PR DESCRIPTION
**Pull request by Daniel-Xu**

### CODE REVIEW

File: engineering/ember.md [hunk 1]
```diff
@@ -120,6 +120,24 @@ this.set('isSelected', true);
 this.get('isSelected');
 ```
 
+### Use brace expansion when the dependent keys are similar
```
Comment (brzpegasus) [inline engineering/ember.md]: "when the dependent keys are properties of the same object" perhaps? "similar" isn't quite right. Or just say "Use brace expansion".

File: engineering/ember.md [hunk 2]
```diff
@@ -120,6 +120,24 @@ this.set('isSelected', true);
...
+
+This allows much less redundancy and easier to read.
```
Comment (brzpegasus) [inline engineering/ember.md]: "is easier to read"

**Comment by brzpegasus** [comment]
LGTM

**Comment by rsocci** [comment]
LGTM

**Pull request merged**